docs/resources/aws_vpc_subnets.md

```diff
@@ -0,0 +1,98 @@
+---
+title: About the aws_vpc_subnets Resource
+---
+
+# aws_vpc_subnets
+
+Use the `aws_vpc_subnets` InSpec audit resource to test properties of some or all subnets.
+
+Subnets are networks within a VPC that can have their own block of IP address's and ACL's.
+VPCs span across all availability zones in AWS, while a subnet in a VPC can only span a single availability zone.
+Separating IP addresses allows for protection if there is a failure in one availability zone.
+
+<br>
+
+## Syntax
+
+An `aws_vpc_subnets` resource block uses an optional filter to select a group of subnets and then tests that group.
+
+    # Test all subnets within a single vpc
+    describe aws_vpc_subnets.where(vpc_id: 'vpc-12345678') do
+      its('subnet_id') { should include ['subnet-12345678', 'subnet-98765432'] }
+    end
+
+<br>
+
+## Examples
+
+The following examples show how to use this InSpec audit resource.
+
+As this is the initial release of `aws_vpc_subnets`, its limited functionality precludes examples.
+
+<br>
+
+## Matchers
+
+### exists
+
+The control will pass if the filter returns at least one result. Use should_not if you expect zero matches.
+
+    # You dont always have subnets, so you can test if there are any.
+    describe aws_vpc_subnets
+      it { should exist }
+    end
+
+    # Test that there are subnets in a vpc
+    describe aws_vpc_subnets.where(vpc_id: 'vpc-12345678')
+      it { should exist }
+    end
+
+## Filter Criteria
+
+### vpc_id
+
+A string identifying the VPC which may or may not contain subnets.
+
+    # Look for all subnts within a vpc.
+    describe aws_vpc_subnets.where( vpc_id: 'vpc-12345678') do
+      its('subnet-ids') { should include ['subnet-12345678', 'subnet-98765432'] }
```

Comment:
The property name in the example has a dash, while the property (below) has an underscore.